x/stakeibc/keeper/hooks.go

```diff
@@ -73,88 +73,39 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInf
 	}
 
 	if epochIdentifier == epochstypes.STRIDE_EPOCH {
-		k.Logger(ctx).Info(fmt.Sprintf("Stride Epoch %d", epochNumber))
+		k.Logger(ctx).Info(fmt.Sprintf("STRIDE EPOCH %d", epochNumber))
```

Comment:
Extremely nit, could we add something like "=============== STRIDE EPOCH %d ===============" here? just makes the epochs easier to spot if we're scrolling through logs!

Reply:
Yeah I was considering the same thing as well! Will add